lib/kernel-manager.coffee

```diff
@@ -69,7 +69,7 @@ class KernelManager
 
         catch e
             unless e.code is 'ENOENT'
-                console.log 'KernelManager: Cannot start existing kernel:\n', e
+                console.error 'KernelManager: Cannot start existing kernel:\n', e
```

Comment:
If now connection file is found the error code would be `ENOENT` and no error will be logged.

Originally we did [throw](https://github.com/nteract/hydrogen/pull/414/files) here. But there were errors coming up caused by [this line](https://github.com/nteract/hydrogen/pull/436/files#diff-aa15d36609a9a564a24156e6778cd62fL60) since this should be fixed now, I think we can log the error again.